addons/website/views/snippets/s_product_list.xml

```diff
@@ -2,55 +2,56 @@
 <odoo>
 
 <template name="Items" id="s_product_list">
+    <t t-set="url" value="''"/>
```

Comment:
At least write it `<t t-set="url" value="url or ''"/>` so this code is there for a reason